video/api/Analyze/Program.cs

```diff
@@ -115,25 +115,25 @@ public static object AnalyzeLabelsGcs(string uri)
         }
 
         // [START analyze_labels]
-        static void PrintLabels(string labelName,
+        static void PrintLabels(string labelName,
```

Comment:
FYI I'm not sure why some of these lines show in the diff, I used VS Code to edit 🤷🏼‍♀️